src/components/Confirm/CaptureViewer/style.css

```diff
@@ -50,6 +50,16 @@
   object-fit: contain;
 }
 
+.croppedDocumentImage {
+  width: 100%;
+  height: 13.5em;
+  object-fit: cover;
```

Comment:
Just FYI for browser compatibility:
- https://caniuse.com/#feat=object-fit
- https://caniuse.com/#feat=object-position

Reply:
To extend on Julius note, it would be good to test this feature on Edge as we had some issue with `object-fit: cover ` support in the past on that browser. See [this line](https://github.com/onfido/onfido-sdk-ui/blob/2dd18a56ba04cc9f247821f21474787efe70b785/src/components/Camera/style.css#L48)

Reply:
Thanks. It only applies for mobiles so I think it is OK. But think I will add a note as a reminder.